### ISSUE TITLE
How Contribute, for adding Persian/Farsi language.for next version?

### ISSUE THREAD

**Issue by mostafazs** [bpellin/keepassdroid#119]
Hi
How can i contributing on translation?
I translate some strings on file, 
https://github.com/bpellin/keepassdroid/blob/master/app/src/main/res/values/strings.xml
I just need to translate this file?

Thanks. Mostafa

**Comment by bpellin** [comment]
Yes, this file contains all of the translatable strings. If you give me a copy of that with Persian/Farsi text, I can add it to the app.
